core/templates/pages/email-dashboard-pages/email-dashboard-page.controller.spec.ts

```diff
@@ -15,6 +15,7 @@
 /**
  * @fileoverview Unit tests for the email dashboard page.
  */
+import { UpgradedServices } from 'services/UpgradedServices';
```

Comment:
Do we need the TODO comment here like in other places?

Reply:
Done